devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

```diff
@@ -71,7 +72,8 @@ metadata {
 
 		// SONOFF
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "SONOFF", model: "BASICZBR3", deviceJoinName: "SONOFF Basic (R3 Zigbee)", ocfDeviceType: "oic.d.smartplug"
-
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "SONOFF", model: "S31 Lite zb", deviceJoinName: "S31 Lite zb", ocfDeviceType: "oic.d.smartplug"
```

Comment:
According to the ticket it looks like this should include SONOF in the deviceJoinName

Reply:
The Vendor just want to show "S31 Lite zb" which is same as their package name.

Reply:
the setupapp name of this device is "S31 Lite zb" ,and I've confirmed with device vendor that whether they want to show the device name in device card as "SONOFF S31 Lite zb" or "S31 Lite zb", and they insist that the name in device card should be "S31 Lite zb"

Reply:
OK, thanks for confirming